nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -169,6 +174,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             return;
         }
 
+        session.getProvenanceReporter().send(flowFile, flowFile.getAttribute(HTTP_REQUEST_URI), stopWatch.getElapsed(TimeUnit.MILLISECONDS));
```

Comment:
@markap14 Would you agree with a URI as below?
< http|https >://< ip client >@<ip server>:< port >/< target >

That would give something like:
http://192.168.1.14@192.168.1.15:/test

This way, the URI could be the same on both HandleHttpRequest and HandleHttpResponse. What do you think?

I can fix URI in both processors using this JIRA (with an update) or open a new one.